crates/node/src/scheduler/mod.rs

```diff
@@ -223,7 +232,117 @@ impl Scheduler {
         Ok(())
     }
 
-    async fn terminate_vm(&self, program: Hash, vm_id: Arc<dyn VMId>) {}
+    async fn reap_zombies(&self) {
+        let mut running_tasks = self.running_tasks.lock().await;
+        let mut running_vms = self.running_vms.lock().await;
+
+        let mut zombies = vec![];
+        for task in running_tasks.iter() {
+            // TODO: Add maximum running time limit for a task here.
```

Comment:
We need to revisit this at some point before devnet launch. I left this as-is for now since I don't have any reasonable guess what's the maximum running time for a task, but here's the place to put it when we know it.